.gitignore

```diff
@@ -3,3 +3,5 @@
 /*.a
 /*.o
 /moz/web-ext-artifacts/
+go.mod
+go.sum
```

Comment:
Was it intentional that a `.gitignore` change got added to this PR?

Reply:
It is from the merge it did.

Reply:
Fixed, updated the branch!